BeeSwift/Gallery/GalleryViewController.swift

```diff
@@ -102,23 +102,21 @@ class GalleryViewController: UIViewController, UICollectionViewDelegateFlowLayou
         }
         
         self.view.addSubview(self.outofdateView)
-        self.outofdateView.backgroundColor = UIColor.Beeminder.gray
         self.outofdateView.snp.makeConstraints { (make) in
             make.right.left.equalTo(0)
             make.top.equalTo(self.deadbeatView.snp.bottom)
             make.height.equalTo(0)
         }
         
         self.outofdateView.addSubview(self.outofdateLabel)
-        self.outofdateLabel.textColor = UIColor.Beeminder.red
+        self.outofdateLabel.textColor = .black
+        self.outofdateLabel.backgroundColor = .Beeminder.red
         self.outofdateLabel.numberOfLines = 0
         self.outofdateLabel.font = UIFont.beeminder.defaultFontHeavy.withSize(12)
         self.outofdateLabel.textAlignment = .center
         self.outofdateLabel.snp.makeConstraints { (make) in
-            make.top.equalTo(3)
-            make.bottom.equalTo(-3)
-            make.left.equalTo(10)
-            make.right.equalTo(-10)
+            make.top.bottom.equalTo(0)
+            make.left.right.equalToSuperview()
```

Comment:
# before
```swift
Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. 
	Try this: 
		(1) look at each constraint and try to figure out which you don't expect; 
		(2) find the code that added the unwanted constraint or constraints and fix it. 
(
    "<SnapKit.LayoutConstraint:0x60000260b7e0@GalleryViewController.swift#109 UIView:0x104c12380.height == 0.0>",
    "<SnapKit.LayoutConstraint:0x60000260ac40@GalleryViewController.swift#118 BeeKit.BSLabel:0x104c12b70.top == UIView:0x104c12380.top + 3.0>",
    "<SnapKit.LayoutConstraint:0x60000260b600@GalleryViewController.swift#119 BeeKit.BSLabel:0x104c12b70.bottom == UIView:0x104c12380.bottom - 3.0>"
)

Will attempt to recover by breaking constraint 
<SnapKit.LayoutConstraint:0x60000260b600@GalleryViewController.swift#119 BeeKit.BSLabel:0x104c12b70.bottom == UIView:0x104c12380.bottom - 3.0>
```


I did not notice the top 3 and bottom -3 padding on the screen from the label to its parent view and removed them thinking it was fine for the label to snug up to its parent.
A case of Chesterton's Fence?

Reply:
My understanding of this warning is
1. We try to hide these warning boxes by setting their height to 0
2. We can't both do that, and also give them padding, as 3+3 > 0 always.

This will be addressed by #542 which sets the views to hidden rather than setting their heights to 0.

Reply:
OK. Maybe I am misunderstanding it.

These 3 and -3 paddings were from out of date label to its containing out of date view. 
The out of date (label containing) view was given an initial height of 0. 

It could be bumped to 42 ein either 
https://github.com/beeminder/BeeSwift/blob/cf9a77490113e1ff7238a9e6b0f043d9f52a8d74/BeeSwift/Gallery/GalleryViewController.swift#L182-L186 or https://github.com/beeminder/BeeSwift/blob/cf9a77490113e1ff7238a9e6b0f043d9f52a8d74/BeeSwift/Gallery/GalleryViewController.swift#L192-L196.

Once it had the height of 42, then the top and bottom paddings of 3 apply.

I think it looks better without the additional space between out of date view and last updated view.
Here with the paddings:
![vertical paddings - dark](https://github.com/user-attachments/assets/027777d6-b578-4403-8dab-1edaece7ff19)
![vertical paddings - light](https://github.com/user-attachments/assets/d09d3936-c7b3-4729-8511-10f8cbe5b42a)

Reply:
I don't want to invest more time here given it goes away with the transition to StackLayout